config/src/main/java/tech/pegasys/pantheon/config/ConfigUtil.java

```diff
@@ -22,4 +23,15 @@ public static OptionalLong getOptionalLong(final JsonObject jsonObject, final St
         ? OptionalLong.of(jsonObject.getLong(key))
         : OptionalLong.empty();
   }
+
+  public static BigInteger getBigInteger(final JsonObject jsonObject, final String key) {
```

Comment:
`Optional<BigInteger>` for consistency with the other method in this class

Reply:
done